x-pack/plugins/apm/server/lib/transactions/charts/get_anomaly_data/index.ts

```diff
@@ -25,14 +26,16 @@ export async function getAnomalySeries({
   setup,
   logger,
   uiFilters,
+  environment,
 }: {
   serviceName: string;
   transactionType: string | undefined;
   transactionName: string | undefined;
   timeSeriesDates: number[];
-  setup: Setup & SetupTimeRange & SetupUIFilters;
+  setup: (Setup & SetupTimeRange) | (Setup & SetupTimeRange & SetupUIFilters);
```

Comment:
I don't think we need to make `SetupUIFilters` optional. Should always default to an empty array afaiu.